dom.go

```diff
@@ -79,11 +79,11 @@ type Restorer interface {
 type HTML struct {
 	Node *js.Object
 
-	tag, text, innerHTML   string
-	styles, dataset        map[string]string
-	properties, attributes map[string]interface{}
-	eventListeners         []*EventListener
-	children               []ComponentOrHTML
+	tag, text, innerHTML, namespace string
```

Comment:
I think `namespace` should be there before `tag`. It goes before tag in the call to `createElementNS`.